packages/default/scss/_variables.scss

```diff
@@ -104,8 +104,10 @@ $toolbar-padding-x: $padding-x !default;
 /// The vertical padding of the container.
 /// @group toolbar
 $toolbar-padding-y: $padding-x !default;
-$toolbar-calc-size: #{$button-calc-size} + #{$toolbar-padding-y * 2} + 2px;
-$toolbar-inner-calc-size: #{$button-calc-size} + #{$toolbar-padding-y * 2};
+$toolbar-button-padding-x: $button-padding-x !default;
+$toolbar-button-padding-y: $button-padding-y !default;
+$toolbar-button-calc-size: #{$form-line-height-em} + #{$toolbar-button-padding-y * 2} + 2px;
```

Comment:
The `2px` at the end of the line appear to be the button border width. If so, this means that they can be substituted by the formula `#{$button-border-width * 2}`. This, in turn, makes the math in the Default theme compatible with the Material theme, meaning that there is no need to override the `$toolbar-button-calc-size` and the `$toolbar-inner-calc-size` variables in Material.